packages/jsActions/nanoflow-actions-native/src/geolocation/GetCurrentLocation.ts

```diff
@@ -35,13 +36,15 @@ export async function GetCurrentLocation(
 ): Promise<mendix.lib.MxObject> {
     // BEGIN USER CODE
 
+    let reactNativeModule: { NativeModules: typeof NativeModules; Platform: typeof Platform } | undefined;
     let geolocationModule: Geolocation | GeolocationStatic | GeolocationServiceStatic;
```

Comment:
I think this should be chained with `| undefined`

Reply:
done 👍